### PR TITLE
Add com.rawtherapee.RawTherapee

### PR DESCRIPTION
**Pull request by hfiguiere**

### CODE REVIEW

**Comment by hfiguiere** [comment]
dupe of pr #789

File: com.rawtherapee.RawTherapee.json [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+  "rename-desktop-file": "rawtherapee.desktop",
+  "rename-icon": "rawtherapee",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc",
+    "--socket=x11",
+    /* Wayland */
+    "--socket=wayland",
+    /* Note playback */
+    "--socket=pulseaudio",
```
Comment (scx) [inline com.rawtherapee.RawTherapee.json]: Audio playback in raw photo processing program?

File: com.rawtherapee.RawTherapee.json [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+    /* MIDI */
+    "--device=all",
```
Comment (scx) [inline com.rawtherapee.RawTherapee.json]: MIDI support in RawTherapee?

File: com.rawtherapee.RawTherapee.json [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+    /* Allow loading, saving files from anywhere (portals don’t work yet) */
+    "--filesystem=home",
+    /* Allow other instances to see lockfiles */
+    "--env=TMPDIR=/var/tmp"
```
Comment (scx) [inline com.rawtherapee.RawTherapee.json]: Did you just copy it from MuseScore?
https://github.com/flathub/org.musescore.MuseScore/blob/41b8fb9fbcb9d2ab83732975488ce5bac08d2448/org.musescore.MuseScore.yaml#L14-L30

File: com.rawtherapee.RawTherapee.json (hunk 1 above)
Comment (hfiguiere) [reply com.rawtherapee.RawTherapee.json]: uh lol. I thought i had cleaned that section up.

File: com.rawtherapee.RawTherapee.json (hunk 3 above)
Comment (hfiguiere) [reply com.rawtherapee.RawTherapee.json]: I copied from the other I did which may have come from MuseScore.

**Comment by hfiguiere** [comment]
Just closing, no point duplicating the effort.

**Pull request closed without merging**